src/utils.rs

```diff
@@ -0,0 +1,36 @@
+use rubato::{
+    Resampler, SincFixedIn, SincInterpolationParameters, SincInterpolationType, WindowFunction,
```

Comment:
I think this was discussed on slack different resamplers have different aliasing characteristics. If you can make sure you pick the most accurate one it'll avoid potential issues at start/end of frame